plugins/scaffolder-backend/api-report.md

```diff
@@ -428,8 +428,18 @@ export class DatabaseTaskStore implements TaskStore {
   // (undocumented)
   heartbeatTask(taskId: string): Promise<void>;
   // (undocumented)
-  list(options: { createdBy?: string; status?: TaskStatus_2 }): Promise<{
+  list(options?: {
+    createdBy?: string | string[];
+    status?: TaskStatus_2 | TaskStatus_2[];
+    limit?: number;
```

Comment:
Maybe we should move `limit` and `offset` into a `pagination` option? Thoughts?

Reply:
No strong opinion either way; do you think there's going to be more pagination options?